pkg/app/cmd.go

```diff
@@ -67,6 +67,9 @@ func runE(
 	opts.bindFlags(kubetest2Flags)
 	artifacts.MustBindFlags(kubetest2Flags)
 
+	if opts.test == "" {
+		cmd.Printf("Running deployer %s version: %s\n", deployerName, shim.GitTag)
```

Comment:
we can log deployer version always, no need to guard it only when `opts.test==""` 
deployer always runs, tester only runs when `opts.test !=""`